infra/project.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "infra",
+  "$schema": "../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "infra/src",
+  "projectType": "application",
+  "generators": {},
+  "targets": {
+    "cli": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "infra",
+        "command": "yarn cli"
+      }
+    },
+    "charts": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "infra",
+        "command": "./scripts/generate-chart-values.sh"
+      }
+    },
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "infra",
+        "command": "yarn build"
+      }
+    },
+    "test": {
+      "executor": "@nx/jest:jest",
+      "options": {
+        "jestConfig": "infra/jest.config.js",
+        "passWithNoTests": true
+      },
+      "outputs": ["{workspaceRoot}/coverage/infra"]
+    },
+    "serve": {},
```

Comment:
~Should just remove the empty targets `serve`, `schemas/codegen` and `docker-native`.

Reply:
I think they need to be available because infra has targets that depend on them, atleast build depends on test.

Will confirm.